callstats-java-sdk/src/main/java/io/callstats/sdk/CallStatsAuthenticator.java

```diff
@@ -239,10 +222,10 @@ public void onResponse(HttpResponse response) {
 							scheduleAuthentication(appId, appSecret, bridgeId,httpClient);
 						}
 					}
-				} else if (responseStatus == GATEWAY_ERROR) {
+				} else if (responseStatus == CallStatsResponseStatus.GATEWAY_ERROR ) {
 					scheduleAuthentication(appId, appSecret, bridgeId,httpClient);
 					listener.onError(CallStatsErrors.APP_CONNECTIVITY_ERROR, "SDK Authentication Error");
-				} else if (responseStatus == INVALID_PROTO_FORMAT_ERROR) {
+				} else if (responseStatus == CallStatsResponseStatus.INVALID_PROTO_FORMAT_ERROR) {
 					listener.onError(CallStatsErrors.AUTH_ERROR, "SDK Authentication Error");
 				} else {
 					listener.onError(CallStatsErrors.HTTP_ERROR, "SDK Authentication Error");
```

Comment:
Do we give up if HTTP error is met? Code should write e-mail to us and probably keep on trying to reconnect.